state-reconstruct-fetcher/src/types/v3.rs

```diff
@@ -3,12 +3,20 @@ use ethers::{
     types::U256,
 };
 use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use tokio::time::{sleep, Duration};
+use zkevm_circuits::eip_4844::ethereum_4844_data_into_zksync_pubdata;
 
 use super::{
     common::{parse_compressed_state_diffs, read_next_n_bytes, ExtractedToken},
-    CommitBlockFormat, CommitBlockInfo, L2ToL1Pubdata, ParseError,
+    L2ToL1Pubdata, ParseError,
 };
-use crate::constants::zksync::L2_TO_L1_LOG_SERIALIZE_SIZE;
+use crate::constants::zksync::{L2_TO_L1_LOG_SERIALIZE_SIZE, PUBDATA_COMMITMENT_SIZE};
+
+/// `MAX_RETRIES` is the maximum number of retries on failed blob retrieval.
+const MAX_RETRIES: u8 = 5;
```

Comment:
As I see it, the values are the same for the ethers provider and blob storage just because we didn't optimize them yet... :-) If the different servers ever require some change to the constant, certainly those will be independent changes...

Reply:
Alright, sure, you've convinced me :smiley: